mwdb/resources/tag.py

```diff
@@ -78,6 +79,9 @@ def get(self):
 
 
 class TagResource(Resource):
+    on_created = hooks.on_created_tag
+    on_reuploaded = hooks.on_reuploaded_tag
+
```

Comment:
Following the previous comment
```suggestion
    on_created = hooks.on_created_tag
    on_reuploaded = hooks.on_reuploaded_tag

```

Reply:
wrong siggestion. resolving